src/Network/Network/PrivateEndpoint/PrivateDnsZoneGroup/PrivateDnsZoneGroupBaseCmdlet.cs

```diff
@@ -37,8 +37,9 @@ public bool IsPrivateDnsZoneGroupPresent(string resourceGroupName, string privat
                 var group = GetPrivateDnsZoneGroup(resourceGroupName, privateEndpointName, privateDnsZoneGroupName);
                 return (group != null);
             }
-            catch (ErrorException exception)
+            catch (Rest.RestException ex) when (ex is ErrorException || ex is Rest.Azure.CloudException)
```

Comment:
@vidai-msft could you provide more context about changing the exception type?

Reply:
The exception thrown by the Network.Management.Sdk is **CloudException** while the original one is **ErrorException**. **RestException** is the base class for both. So the purpose is to capture both ErrorException and CloudException and then ignore if the response code is 404.